teacher.rb

```diff
@@ -1,10 +1,8 @@
 require_relative 'person'
 
 class Teacher < Person
-  attr_reader :specialization
-
-  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  def initialize(age, specialization, name = 'Unknown')
+    super(age: age, name: name)
     @specialization = specialization
   end
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add some more meaningful features in the Key Features section to make your readme descriptive and professional 👍